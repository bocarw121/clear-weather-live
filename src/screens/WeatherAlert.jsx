import React from "react";
import { Text } from "react-native";

import LinearGradientView from "../components/LinearGradientView";

const WeatherAlert = () => {
  return (
    <LinearGradientView>
      <Text>Weather Alerts</Text>
    </LinearGradientView>
  );
};

export default WeatherAlert;
