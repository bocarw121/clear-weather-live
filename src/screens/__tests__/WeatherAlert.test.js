import React from "react";

import { render } from "@testing-library/react-native";

import WeatherAlert from "../WeatherAlert";

it("should render the current screen", () => {
  const screen = render(<WeatherAlert />);

  expect(screen).toMatchSnapshot();
});
