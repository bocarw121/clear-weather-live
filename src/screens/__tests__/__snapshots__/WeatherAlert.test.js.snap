// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the current screen 1`] = `
<View>
  <ViewManagerAdapter_ExpoLinearGradient
    proxiedProperties={
      Object {
        "colors": Array [
          4278224127,
          4280070121,
        ],
        "endPoint": Array [
          1,
          1,
        ],
        "locations": undefined,
        "startPoint": Array [
          0,
          0,
        ],
      }
    }
    style={
      Array [
        Object {
          "height": "100%",
        },
      ]
    }
  >
    <RCTScrollView>
      <View>
        <Text>
          Weather Alerts
        </Text>
      </View>
    </RCTScrollView>
  </ViewManagerAdapter_ExpoLinearGradient>
</View>
`;
